From dc6b9371110febac2cd49e7284ad238262014a46 Mon Sep 17 00:00:00 2001
From: Peter Felecan <pfelecan@opencsw.org>
Date: Mon, 22 Jul 2013 10:53:42 +0200
Subject: [PATCH] manage previous site package

this is to support the transition between an unversiond installation tree
---
 Lib/site.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Lib/site.py b/Lib/site.py
index f1b0ae8..04fb5f2 100644
--- a/Lib/site.py
+++ b/Lib/site.py
@@ -292,6 +292,7 @@ def getsitepackages():
                                         "python" + sys.version[:3],
                                         "site-packages"))
             sitepackages.append(os.path.join(prefix, "lib", "site-python"))
+            sitepackages.append(os.path.join(prefix, "lib", "python", "site-packages"))
         else:
             sitepackages.append(prefix)
             sitepackages.append(os.path.join(prefix, "lib", "site-packages"))
-- 
1.8.3.1

